samples/update_protection_runs/update_job_run.py

```diff
@@ -0,0 +1,133 @@
+# Copyright 2019 Cohesity Inc.
+#
+# Python example to extend the snapshot retention per protection job run. The
+# runs can be filtered by start/end date.
+#
+# Usage : python update_job_run.py --job_name=<protect-job>
+# --start_time=2019-2-31 --end_time=2019-7-29 --extend=<num-of-days>
```

Comment:
--start_time must be --start_date same with --end_time.

Reply:
Ack, good catch.